.github/workflows/ci.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - release-*
```

Comment:
This should run for `release-*` branches as well to build an image which will be promoted by the "Create Release" workflow.